dask/array/reductions.py

```diff
@@ -423,12 +428,20 @@ def min(a, axis=None, keepdims=False, split_every=None, out=None):
     )
 
 
+def chunk_min(x, axis=None, keepdims=None, *args, **kwargs):
```

Comment:
From what I can tell it doesn't look like we're ever passing any `*args` or `**kwargs` here (from the too-level `min` method where this is function called). If that's the case, I'd prefer to remove `*args` and `**kwargs` here if we're not able to use them